packages/web-runtime/src/services/auth/abilities.ts

```diff
@@ -39,7 +39,7 @@ export const getAbilities = (
       { action: 'update-all', subject: 'Space' }
     ],
     'list-all-spaces.all': [{ action: 'read-all', subject: 'Space' }],
-    'set-space-quota.all': [{ action: 'set-quota-all', subject: 'Space' }]
+    'Drive.ReadWriteQuota.Project.all': [{ action: 'set-quota-project', subject: 'Space' }]
```

Comment:
It got me thinking: shouldn't the action be the same like before: `set-quota-all`? Because `project` is already covered by the subject `Space` here. And `-all` is coming from the server, hence we should add it as well.

Reply:
hmmm.... for now i'm fine using `set-quota-all`... but if we get a second type then we need to change it.
The backend decided to have it explicit `Drive.ReadWriteQuota.Project.all`, than we should too.

Fine using `set-quota-project-all`, what do you think?